src/app/src/actions/facilityListDetails.js

```diff
@@ -201,20 +201,40 @@ export function rejectFacilityListItemMatch(facilityMatchID) {
             );
         }
 
-        return apiRequest
-            .post(createRejectFacilityListItemMatchURL(facilityMatchID))
-            .then(({ data }) =>
-                dispatch(completeRejectFacilityListItemPotentialMatch(data)),
-            )
-            .catch(err =>
-                dispatch(
-                    logErrorAndDispatchFailure(
-                        err,
-                        'An error prevented rejecting that match',
-                        failRejectFacilityListItemPotentialMatch,
-                    ),
+        let chain = Promise.resolve();
+
+        facilityMatchIDs.forEach(facilityMatchID => {
+            chain = chain.then(() =>
+                apiRequest
+                    .post(createRejectFacilityListItemMatchURL(facilityMatchID))
```

Comment:
As I understand you didn’t modify or create new API endpoint, just create a chain and reject facility match one by one?

Reply:
Yes, correct.